docker/docker-compose-build.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.6"
+
+services:
+  node-webpack:
+    image: node:16
+    volumes:
+      - ..:/root/project
+    working_dir: /root/project
+    command: ["sh", "-c", "npm install -g npm-check-updates && ./compile-front-end-packages.sh -ui" ]
```

Comment:
Given that it would look like:
```yaml
command:
  - sh
  - -c
  - |
    npm install -g npm-check-updates && \
    ./compile-front-end-packages.sh -ui
```

I thought the above was clearer and consistent with the rest of the services in `docker-compose-build.yml`.

Reply:
Yes, I reconsidered it and I do agree with you. Thanks for your time